common/src/main/java/org/apache/cassandra/sidecar/common/ICassandraAdapter.java

```diff
@@ -28,6 +26,6 @@
  */
 public interface ICassandraAdapter
 {
-    List<NodeStatus> getStatus();
+    NodeSettings getStatus();
```

Comment:
Now this method won't become getSettings() ?

Probably, this could be something like this:

```
boolean getStatus();
NodeSettings getSettings();
```

Reply:
or just `settings()` following Cassandra conventions